athenahealth/patients.go

```diff
@@ -355,6 +358,14 @@ func (h *HTTPClient) UpdatePatient(ctx context.Context, patientID string, opts *
 			form.Add("assignedsexatbirth", *opts.AssignedSexAtBirth)
 		}
 
+		if opts.ConsentToCall != nil {
+			form.Add("consenttocall", strconv.FormatBool(*opts.ConsentToCall))
```

Comment:
Do these need to be strings even though the API docs describe them as boolean?

Reply:
`url.Values` only accepts string values
this does seem to be working but i think there is an issue with what athena api is documenting.. it seems like a combination of `hasmobile` and `hasmobileyn` is accepted depending on whether getting or updating a patient.

Reply:
After testing - i confirmed that the PUT patient endpoint expects `hasmobileyn` while the GET patient endpoint send `hasmobile`

Reply:
🤘